crates/db-allocator/src/interval_tree.rs

```diff
@@ -602,16 +601,22 @@ impl<T> IntervalTree<T> {
     /// # Examples
     /// ```rust
     /// extern crate db_allocator;
-    /// use db_allocator::{IntervalTree, Range, NodeState};
+    /// use db_allocator::{IntervalTree, NodeState, Range};
```

Comment:
Thanks for the format work:)
Could you please use a new pull request to do this?

Reply:
Sure, I will create a new pr later.